doc/development.rst

```diff
@@ -37,6 +37,164 @@ The source repository (and current branch) of each of these packages
 in use is determined by the file
 `requirements.txt in mirgecom <https://github.com/illinois-ceesd/mirgecom/blob/master/requirements.txt>`__.
 
+
+Working with Pull Requests
+--------------------------
+
+We are using GitHub's pull requests (PRs) feature to integrate changes into
+mirgecom. Pull requests are based on git branches that are merged into another
+branch (usually the master branch). Note that pull requests are a GitHub
+feature and live outside the main git functionality; the `git` program itself
+has no knowledge of them.
```

Comment:
Mention that `conda install git` can be used to install git. (or that emirge automatically installs it?)